apps/wallet-mobile/src/features/Exchange/common/constants.ts

```diff
@@ -1,2 +1,15 @@
+import {Exchange} from '@yoroi/types'
+
+import {BanxaLogo} from '../illustrations/BanxaLogo'
+import {EncryptusLogo} from '../illustrations/EncryptusLogo'
+
 export const storageRootExchange = 'ramp-on-off'
```

Comment:
yap, it will pop up the warning again, but I think it worth it, to keep consistency
```suggestion
export const storageRootExchange = 'exchange'
```